pkgs/development/python-modules/pylibftdi/default.nix

```
@@ -0,0 +1,52 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, pythonAtLeast
+, httpserver
+, libftdi1
+, libusb1
+}:
+
+buildPythonPackage rec {
+  pname = "pylibftdi";
+  version = "0.17.0";
+  pythonAtLeast = "3"; # compatibility with httpserver
```

I think you want,
```suggestion
  disabled = pythonOlder "3"; # compatibility with httpserver
```
 pythonAtLeast is a function of shape `string -> bool`